configs/components/libxslt.rb

```diff
@@ -15,6 +15,8 @@
 
     # libxslt is picky about manually specifying the build host
     build = "--build x86_64-linux-gnu"
+    # don't depend on libgcrypto
+    disable_crypto = "--without-crypto"
```

Comment:
Without this flag, the build fails because the configure script finds the libgcrypto header file from the x86_64 environment and tries to build and link to it.